source/_templates/search.html

```diff
@@ -0,0 +1,73 @@
+{# Import the theme's layout. #}
+{% extends "!layout.html" %}
+
+{# Custom CSS overrides #}
+
+{% block extrahead %}
+   {%- include "head.html" %}
+   {{ super() }}
+{% endblock %}
+
+{# Override content block #}
+
+{%- macro miniosidebar() %}
+   {%- if render_sidebar %}
+      <nav
+         class="docs"
+         role="navigation"
+      >
+         {%- if sidebars != None %}
+            {#- new style sidebar: explicitly include/exclude templates #}
+            {%- for sidebartemplate in sidebars %}
+               {%- include sidebartemplate %}
+            {%- endfor %}
+         {%- endif %}
+      </nav>
+   {%- endif %}
+{%- endmacro %}
+
+{%- block header %}
+   {%- include "header.html" %}
+{%- endblock %}
+
+{%- block content %}
+   <section class="content">
+      <div class="container">
+         <div class="sidebar inactive scrollbar">
+            <div class="hide-aside visible-rm">
+               <button type="button" class="icon">
+                     {%- include "icons/close.html" %}
+                     Close Doc Navigation
+               </button>
+            </div>
+
+            <a class="sidebar__title" href="{{ pathto('index') }}">{{ shorttitle}}</a>
+            
+            {{ miniosidebar() }}
+         </div>
+
+         {%- include "toc.html" %}
+
+         <div class="content__main" id="search-documentation">
+            {% block body %} 
+               <div id="search">
```

Comment:
Can we add a page title here?

# Search

or

# Docs Search

Reply:
Sure @djwfy, I will send another to add the title.